test/src/util/request_test_suite.c

```diff
@@ -18,7 +18,7 @@
 
 #include "sgx_urts.h"
 #include "pelz_enclave.h"
-#include "pelz_enclave_u.h"
+#include ENCLAVE_HEADER_UNTRUSTED
```

Comment:
The test files should probably explicitly include ```pelz_enclave_*.h``` since that's what they always use.